### PR TITLE
Update release-notes.md

### PR DESCRIPTION
**Pull request by kellybos**
Update about diagnostic log change.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/release-notes/release-notes.md [hunk 1]
```diff
@@ -22,6 +22,10 @@ description: Release notes for Microsoft Teams admin features.
 
 These release notes provide information about new admin features for Microsoft Teams.
 
+## June 1, 2021
+
+Collection of Teams diagnostic logs just got easier. By right-clicking on the Teams icon in the system tray and choosing **Collect support files** all desktop and debug logs will be collected in one folder in Downloads. This option will also include media logs (if enabled in Settings). More information can be found at https://docs.microsoft.com/en-us/microsoftteams/log-files.
```
Comment (JohanFreelancer9) [inline Teams/release-notes/release-notes.md:27]:
```suggestion
Collection of Teams diagnostic logs just got easier. By right-clicking on the Teams icon in the system tray and choosing **Collect support files** all desktop and debug logs will be collected in one folder in Downloads. This option will also include media logs (if enabled in Settings). For more information, see [Use log files to monitor and troubleshoot Microsoft Teams](/microsoftteams/log-files).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @kellybos, could you please commit my suggested changes if you agree with them. I have suggested changing the reference sentence to be consistent with the other updates. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by kellybos** [comment]
@tonysmit Any update on the final review?

**Comment by scanum** [comment]
@tonysmit Hope all is well. Wanted to follow up here. Thanks.

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@CarolynRowe 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by CarolynRowe** [comment]
I do not own release notes.  Please follow up with Tony Smith.

**Comment by tiburd** [comment]
Closing due to age and file conflict. Menu path isn't accurate in new Teams.

cc: @scanum

**Pull request closed without merging**